spec/features/user_groups_spec.rb

```diff
@@ -22,7 +22,10 @@
     org = create(:user_group)
     login_as(admin)
 
-    visit edit_user_group_path(org)
+    visit user_groups_path
+    find("a[aria-label='#{t('links.aria.edit', name: org.name)}']").click
+
+    expect(current_path).to eq(edit_user_group_path(org))
```

Comment:
I don't know if you even need this expectation. The fact that the following steps work means that you are on the correct page (because there are name and description form fields). what do you think?